apps/fabric-website/src/pages/HomePage/HomePage.styles.ts

```diff
@@ -404,11 +404,9 @@ export const getStyles = (props: IHomePageStyleProps): IHomePageStyles => {
             color: theme.palette.white
           },
 
-          [`
-            &:not(.is-disabled):hover .${classNames.linkText},
-            &:not(.is-disabled):active .${classNames.linkText},
-            &:not(.is-disabled):active:hover .${classNames.linkText}
-          `]: {
+          [`&:not(.is-disabled):hover .${classNames.linkText}, ` +
```

Comment:
was this broken before? is this just a prettier change?

Reply:
Functionally it worked fine, but it will result in a string literal like this which can't be minified further:
```
&:not(.is-disabled):hover .${classNames.linkText},\n            &:not(.is-disabled):active .${classNames.linkText},\n            &:not(.is-disabled):active:hover .${classNames.linkText}\n          
```